test/OpenTelemetry.Instrumentation.Http.Tests/HttpClientTests.cs

```diff
@@ -139,7 +139,7 @@ public async Task HttpOutCallsAreCollectedSuccessfullyAsync(HttpTestData.HttpOut
 
                 var metric = requestMetrics[0];
                 Assert.NotNull(metric);
-                Assert.True(metric.MetricType == MetricType.Histogram);
+                Assert.True(metric.MetricType == MetricType.HistogramWithMinMax);
```

Comment:
Since we are changing the MetricType, any exporters which rely on this will be broken. For example: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/MetricItemExtensions.cs#L241
will stop exporting Histograms.

We need to hotfix the OTLP Exporter to use the IsHistogram() check instead. (prometheus already handles this correctly. So is ConsoleExporter..)

Reply:
We will continue to use `MetricType.Histogram` to not break exporters, and only when min and max are not the default +inf and -inf respectively will they be exported.